api/envoy/service/trace/v2/BUILD

```diff
@@ -5,7 +5,7 @@ licenses(["notice"])  # Apache 2
 api_proto_library_internal(
     name = "trace_service",
     srcs = ["trace_service.proto"],
-    has_services = 1,
+    has_services = 0,  # A go-only service due to the external dependency @opencensus_proto//opencensus/proto/trace/v1:trace_proto_go
```

Comment:
What is a go-only service? Aren't all services reasonably implemented in all languages?

Reply:
Go-only in the sense that it has external language-specific dependencies that have only been mapped for Go and not C++ or python or ...

I can update the comment to make that clearer and perhaps leave a `TODO` for good measure.

Reply:
Sounds good. LGTM modulo this bit. Thanks!